### ISSUE TITLE
Fallback to `name` if blockSource attribute is not defined on component

### ISSUE THREAD

**Issue by monachilada** [verstaerker/vue-bem#2]
Great plugin, thanks!

It would be great if the plugin was provided with a blockSource property, but then that property was not found on the component, that it would automatically fallback to the default `name` property.

Is there already a good way to do this when installing the plugin globally?

**Comment by patric-eberle** [comment]
Sorry @monachilada, I did not see that you added a ticket... A blockSource setting is already available, but looks like I forgot to document it.

By fallback do you mean per component or on global scope? On global it should already wokr. If you don‘t define a value in your options it will usethe „name“ fallback.

**Comment by monachilada** [comment]
Yes, I figured out the blockSource setting from digging through the code. But it's not quite what I meant. Let's say I'm doing something like this globally:

```js
import vueBem from '@verstaerker/vue-bem';

Vue.use(vueBem, {
  blockSource: 'blockName',
});
```

If I set `blockName: 'blah'` in my component, I'm all good. But if it's not defined, I'd love it to fallback to `name` as normal.

Use case: 90% of my components will have the same block name as the component name, however one or two should be simplified.

For example I would like the block name for a component called `'base-button'` to simply be `'button'`.
